tests/unittests/test_dftb_feeds_periodic.py

```diff
@@ -202,9 +202,14 @@ def test_scipyskfeed_pbc_batch(device, skf_file: str):
 # tbmalt.physics.dftb.feeds.SkFeed #
 #########################################
 
+
 # Single
 def test_skffeed_pbc_single(device, skf_file: str):
     """SkFeed matrix single system operability tolerance test"""
+
+    # Hmmmm... for some reason the tolerance checks pass when run via PyTest
+    # but fail if run manually.
```

Comment:
I also test it on my laptop manually, but it can pass from my side. However, actually the current setting of `rtol=1e-05, atol=1e-08` sometimes give a big overall tolerance of ~1E-4, for the H2 case for example, computing by the formula `atol+rtol×∣other∣`. We may change the setting to `rtol=1e-08, atol=1e-08`.